sys-cluster/teleport/files/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- src/github.com/gravitational/teleport/Makefile
```

Comment:
This is a bad patch name. We usually prefix patches with `${P}-`, especially if they're likely to apply to one specific version only. It makes the updates and cleanup easier.